examples/ota-provider-app/esp32/README.md

```diff
@@ -0,0 +1,53 @@
+# ota-provider-app (ESP32)
+
+### Build and flash OTA apps
+
+-   Build and flash [ESP32 OTA provider app](.) and
+    [ESP32 OTA Requestor app](../../ota-requestor-app)
+
+```
+$ cd examples/ota-provider-app/esp32
+$ idf.py -p <RequestorSerialPort> build flash
+
+$ cd examples/ota-requestor-app/esp32
+$ idf.py -p <ProviderSerialPort> build flash
+```
+
+-   Flash hello-world OTA binary in providor app. Please find hello-world.bin
```

Comment:
Typo: provider

Should we mention that, this image is the OTA upgrade image will be sent to the provider?